gulp-tasks/reference-docs/filter-tags-to-build.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Please add a JSDoc description to this file, include at least `@fileoverview` and `@author`

Reply:
Done.